### DISCUSSION TITLE
logo in the header and the space between homeinfo and the first post

### DISCUSSION THREAD

**Discussion by 1mfaasj** [adityatelange/hugo-PaperMod#547] [Q&A] [answered]
I'm wondering how to add an image/logo to the header? see: https://ibb.co/pzScvKc
and how I change the space between homeInfoParams and the first Post, see other screenshot: https://ibb.co/pfm44vD

thanks in advance

**Comment by pixelfabs** [comment, marked as answer]
> I'm wondering how to add an image/logo to the header? see: https://ibb.co/pzScvKc

That's quite a task. The section you're referring to is not as simple as it seems. It holds the theme toggle switch, links you can define via config file (menu: whatever) and language switch links. It uses CSS flex box which adjust depending on screen size. My advice: just don't, You add more than 3 items, it doesn't look great on mobile devices. Remove stuff, it breaks. The theme toggle switch for example is 2 parts, one part in `header.html` and the other in `footer.html`

I made a new flex box container underneath the default one and while it takes a bit more space, it definitely doesn't break things and far easier to update.

> and how I change the space between homeInfoParams and the first Post, see other screenshot: https://ibb.co/pfm44vD

`margin: var(--gap)0 calc(var(--gap) * 2);` under `.first-entry` in `post-entry.css`

I don't use it, but I assume if you disable it or play around with the "2" you will see some results.
